src/components/container/Goal/Goal.css

```diff
@@ -0,0 +1,63 @@
+.mdhui-goal {
+    display: grid;
+    grid-template-columns: max-content auto;
+    padding: 8px 16px;
```

Comment:
Did this come from a design? It does not look balanced in the top corner in the story.

Reply:
I've switched to `margin` and used a value of `16px` for both the sides and top.  This gives it better balance with respect to the whitespace on the sides and top without adding too much space between sequential goal components.